solidity/contracts/deposit/DepositUtils.sol

```diff
@@ -44,6 +44,9 @@ library DepositUtils {
         uint256 courtesyCallInitiated; // When the courtesy call is issued
         address payable liquidationInitiator;
 
+        // SET SOLELY IF FUNDER ABORT IS REQUESTED
+        bytes funderAbortOutputScript; // An output script in case of funder abort
```

Comment:
Should we use storage here or rely on events?

Reply:
Asked myself same… Feels weird to only rely on events given how we run the rest of things, but there's also no need to confirm anything about the script later, so we could do without storage…

Reply:
Switching to events-only here. Also making this actually emit the event instead of just pretending 😬